### PR TITLE
refactor(frontend): update layout of import page

### PR DESCRIPTION
**Pull request by kelsos**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/pages/import/index.vue [hunk 1]
```diff
@@ -3,26 +3,31 @@ const { t } = useI18n();
...
-          </ExternalLink>
-        </template>
-      </i18n>
-    </RuiAlert>
+  <div class="p-4">
+    <TablePageLayout :title="[t('import_data.title')]">
+      <template #buttons>
+        <HintMenuIcon>
+          {{ t('import_data.description') }}
+        </HintMenuIcon>
```
Comment (lukicenturi) [inline frontend/app/src/pages/import/index.vue:11]: image

The menu is too big, maybe you can set max-width there?

Comment (kelsos) [reply frontend/app/src/pages/import/index.vue:11]: Let me do some changes there

**Comment by kelsos** [comment]
@lukicenturi I modified the component to add a default max-width so now it should be ok

**Review by lukicenturi — APPROVED** [review]
LGTM!

**Pull request merged**